src/test/resources/pacts/publicapi-connector-search-payment-with-charge-in-awaiting-capture-state.json

```diff
@@ -0,0 +1,185 @@
+{
+  "consumer": {
+    "name": "publicapi"
+  },
+  "provider": {
+    "name": "connector"
+  },
+  "interactions": [
+    {
+      "description": "search a charge request with awaiting charge request state",
+      "providerStates": [
+        {
+          "name": "a charge with delayed capture true and awaiting capture request status exists",
+          "params": {
+            "gateway_account_id": "123456",
+            "charge_id": "ch_123abc456def"
+          }
+        }
+      ],
+      "request": {
+        "method": "GET",
+        "path": "/v1/api/accounts/123456/charges",
+        "query": {
+          "transactionType": ["charge"]
+        }
+      },
+      "response": {
+        "status": 200,
+        "headers": {
+          "Content-Type": "application/json"
+        },
+        "body": {
+          "page": 1,
+          "total": 1,
+          "count": 1,
+          "results": [
+            {
+              "amount": 100,
+              "state": {
+                "finished": false,
+                "status": "submitted"
+              },
+              "description": "Test description",
+              "reference": "aReference",
+              "language": "en",
+              "links": [
+                {
+                  "rel": "self",
+                  "method": "GET",
+                  "href": "https://connector/v1/api/accounts/123456/charges/ch_123abc456def"
+                },
+                {
+                  "rel": "refunds",
+                  "method": "GET",
+                  "href": "https://connector/v1/api/accounts/123456/charges/ch_123abc456def/refunds"
```

Comment:
Looks like we are using different hosts for connector in one file, if you would like you can change it to one variant. They are ignored anyway, because we have matchers, but just to be consistent within one file